### PR TITLE
 Fix minor issues, refactoring

### PR DESCRIPTION
**Pull request by Yasara123**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/SiddhiAppMetricsHistory.java [hunk 1]
```diff
@@ -100,7 +99,7 @@ public String getThroughputRecent() {
 
     public void setThroughputRecent(List<List<Object>> throughput) {
         if ((throughput != null) && (!throughput.isEmpty())) {
-            this.throughputRecent = NumberFormat.getIntegerInstance().format((Long) (throughput.get(throughput.size
+            this.throughputRecent = NumberFormat.getIntegerInstance().format((throughput.get(throughput.size
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/SiddhiAppMetricsHistory.java]: Better to split the line such a way the readability is improved

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java [hunk 2]
```diff
@@ -527,13 +527,13 @@ public List selectWorkerThroughput(String workerId, long timeInterval, long curr
         String resolvedSelectWorkerThroughputQuery = resolveTableName(selectWorkerThroughputQuery,
                 "METRIC_METER");
         String resolvedQuery = resolvedSelectWorkerThroughputQuery.replace(SQLConstants.PLACEHOLDER_COLUMNS,
-                "AVG(result.COUNT)").replace
+                "AVG(result.M1_RATE)").replace
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: Break lines at dots (.) not near parenthesis

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java [hunk 3]
```diff
@@ -549,14 +549,14 @@ public List selectWorkerAggregatedThroughput(String workerId, long timeInterval,
         String resolvedSelectWorkerThroughputQuery = resolveTableName(selectWorkerAggregatedThroughputQuery,
                 "METRIC_METER");
         String resolvedQuery = resolvedSelectWorkerThroughputQuery.replace(SQLConstants.PLACEHOLDER_COLUMNS,
-                "AVG(result.COUNT)").replace
+                "AVG(result.M1_RATE)").replace
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: break lines at dots

File: components/org.wso2.carbon.status.dashboard.web/src/siddhi-app/AppSpecific.jsx [hunk 4]
```diff
@@ -237,7 +237,7 @@ export default class WorkerSpecific extends React.Component {
     renderLatencyChart() {
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.web/src/siddhi-app/AppSpecific.jsx]: All jsx files in this PR need to be reformatted. hard coded styling need to be moved to css files

Comment (erangatl) [reply components/org.wso2.carbon.status.dashboard.web/src/siddhi-app/AppSpecific.jsx]: It's better to pass styles through a const, so the code looks nice. Not recommended to move MUI styles to CSS.

**Pull request merged**